### ISSUE TITLE
md5 is not match

### ISSUE THREAD

**Issue by The-Daruz** [Gictorbit/illustratorCClinux#35]
curl: (18) transfer closed with 369372385 bytes remaining to read
error: sorry something went wrong during download illustratorCC17.tgz

Help, how can I solve this problem? :(

**Comment by RonakonGit** [comment]
any updates ?

**Comment by The-Daruz** [comment]
no, but somewhere else I managed to install it correctly, I leave you the link of the page: https://rtvmundo.com/cursos-gratuitos/descargar-adobe-illustrator-cs6-para-pc-version-ultima/

**Comment by ViSHNUPrABU** [comment]
illustratorCC17.tgz downloaded by commenting the below two lines in illustratorCClinux/scripts/sharedFuncs.sh :

ariapkg=$(package_installed aria2c "summary")                    ----> 196
curlpkg=$(package_installed curl "summary")                        ----> 197

Then , the file downloaded in three tries.
After that I faced a problem --> cp: cannot create regular file '/home/root/.local/share/applications/illustratorCC.desktop': No such file or directory error: can't copy desktop entry.

For this problem we are gonna manually copy the desktop entry to the global desktop entry location.
For this just open the file manager in root mode( Just type in terminal -> 'sudo nautilus'  ) , then the file manager opens up.

The Icon will not be detected , so for that I copied the entire path of the AiIcon.png in illustratorCClinux/images and pasted it in the illustratorCClinux/scripts/illustratorCC.desktop file in the 8th line 'Icon=$copied_path'

Lastly I copied my illustratorCC.desktop in illustratorCClinux/scripts and pasted it in /usr/share/applications.

Then the illustrator added to the desktop entry.

**Comment by samarahmad247** [comment]
thanx and its working fine for me , just luanch the application through gtk-launch <file>

**Comment by sarwar-jahin-port** [comment]
> illustratorCC17.tgz downloaded by commenting the below two lines in illustratorCClinux/scripts/sharedFuncs.sh :
> 
> ariapkg=$(package_installed aria2c "summary") ----> 196 curlpkg=$(package_installed curl "summary") ----> 197
> 
> Then , the file downloaded in three tries. After that I faced a problem --> cp: cannot create regular file '/home/root/.local/share/applications/illustratorCC.desktop': No such file or directory error: can't copy desktop entry.
> 
> For this problem we are gonna manually copy the desktop entry to the global desktop entry location. For this just open the file manager in root mode( Just type in terminal -> 'sudo nautilus' ) , then the file manager opens up.
> 
> The Icon will not be detected , so for that I copied the entire path of the AiIcon.png in illustratorCClinux/images and pasted it in the illustratorCClinux/scripts/illustratorCC.desktop file in the 8th line 'Icon=$copied_path'
> 
> Lastly I copied my illustratorCC.desktop in illustratorCClinux/scripts and pasted it in /usr/share/applications.
> 
> Then the illustrator added to the desktop entry.

It been almost 24hours I was trying to install illustrator. Finally, this method worked took 12 try to download. Thank you so much.
